packages/keycloak-user-management/src/components/CreateEditUser/index.tsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import { Col, notification, Row } from 'antd';
+import { RouteComponentProps } from 'react-router';
+import * as Yup from 'yup';
+import { Store } from 'redux';
+import { connect } from 'react-redux';
+import reducerRegistry from '@onaio/redux-reducer-registry';
+import { HeaderBreadCrumb } from '../HeaderBreadCrumb';
+import {
+  fetchKeycloakUsers,
+  KeycloakUser,
+  makeKeycloakUsersSelector,
+  reducer as keycloakUsersReducer,
+  reducerName as keycloakUsersReducerName,
```

Comment:
All the keycloak user store and reducers need to be moved from the global store package and into this keycloak user package.

Reply:
Tracking this via https://github.com/OpenSRP/web/issues/64